sources/packages/web/src/views/student/financial-aid-application/Applications.vue

```diff
@@ -20,7 +20,8 @@
         $router.push({
           name: StudentRoutesConst.DYNAMIC_FINANCIAL_APP_FORM,
           params: {
-            selectedForm: formName,
+            selectedForm: formName.formName,
```

Comment:
Please consider renaming the variable name to something like "programYear", since it no longer represents the formName.